docs/resources/device_settings_policy.md

```diff
@@ -53,7 +53,7 @@ resource "cloudflare_device_settings_policy" "developer_warp_policy" {
 - `exclude_office_ips` (Boolean) Whether to add Microsoft IPs to split tunnel exclusions.
 - `match` (String) Wirefilter expression to match a device against when evaluating whether this policy should take effect for that device.
 - `precedence` (Number) The precedence of the policy. Lower values indicate higher precedence.
-- `service_mode_v2_mode` (String) The service mode. Defaults to `warp`.
+- `service_mode_v2_mode` (String) The service mode. Defaults to `warp`. A service mode is any of the following values: `1dot1`, `warp`, `proxy`, `posture_only`, `warp_tunnel_only`
```

Comment:
this will need to be added to the [schema definition](https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/sdkv2provider/schema_cloudflare_device_settings_policy.go#L83) to avoid being overwritten on next run.

you can use the `renderAvailableDocumentationValuesStringSlice` helper to format this properly too (example: https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/sdkv2provider/schema_cloudflare_record.go#L43)

Reply:
Good to know, thanks for the heads up.